src/blocks/sound.rs

```diff
@@ -115,6 +118,9 @@ pub struct SoundConfig {
     /// The steps volume is in/decreased for the selected audio device (When greater than 50 it gets limited to 50)
     #[serde(default = "SoundConfig::default_step_width")]
     pub step_width: u32,
+
+    #[serde(default = "SoundConfig::default_on_clicked")]
+    pub on_clicked: Option<String>,
```

Comment:
Can you change this to `on_click`, for consistency with the [Custom block](https://github.com/greshake/i3status-rust/blob/master/blocks.md#custom)?